tutorials/introductory/usage.py

```diff
@@ -452,7 +465,7 @@ def my_plotter(ax, data1, data2, param_dict):
 # .. _PyQt5: https://riverbankcomputing.com/software/pyqt/intro
 #
 # WX backends
-# ===========
+# -----------
 #
 # At present the release version of `wxPython` (also known as wxPython classic)
 # does not support python3. A work in progress redesigned version known as
```

Comment:
Is this statement still true? wxpython 4.0.0b1 seems to support py3k. I don't know when the support started, though.

Reply:
wxpython 4.0 is the same as Phonix which is the first version to support python 3. We could update this to point to wx 4.0 beta